Bunny Prisoner Locator.cpp

```diff
@@ -0,0 +1,18 @@
+#include<iostream>

```

Comment:
put your solutions in a separate directory named "alternate solutions by [your name].